routes/owners/api.js

```diff
@@ -68,7 +68,7 @@ export function index(req, res) {
     return body && body.pull_request && prActionTypes.indexOf(body.action) > -1;
   }
 
-  function isReviewAction() {
+  function isReviewAction(body) {
```

Comment:
Curious: Was this function using a different `body` earlier? Or is this a bug that got fixed?

Reply:
because of js closures, its the same `body` from the outer scope, i just wanted to be explicit about passing it in.